### PR TITLE
fix: build script on windows environments

### PR DESCRIPTION
**Pull request by userquin**
Copying `README.md` on `unocss` and `LICENSE` on `vscode` breaks windows build.

### CODE REVIEW

**Comment by userquin** [comment]
upps, `LICENSE` not being copied.

**Comment by johannschopplich** [comment]
@userquin What about using [shx](https://github.com/shelljs/shx)? That way the shell scripts can stay readable and no extra scripts would have to be introduced.

Example usage: `shx cp ../../README.md .`

File: packages/unocss/scripts/copy-files.ts [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import { copyFileSync } from 'fs'
+import { resolve } from 'path'
+
+// relative to scripts directory
```
Comment (antfu) [inline packages/unocss/scripts/copy-files.ts:4]: Could we move this to the top level so we copy them once on build

Comment (userquin) [reply packages/unocss/scripts/copy-files.ts:4]: sure, to root script directory

**Comment by userquin** [comment]
@johannschopplich we should this way to allow copying multiple files at once, and we can reuse the module for ohter things when required

**Pull request merged**